src/nut.c

```diff
@@ -98,9 +101,51 @@ static int nut_add_ups(const char *name) {
   return (0);
 } /* int nut_add_ups */
 
+static int nut_force_ssl(const char *value){
+  if (strcasecmp(value, "true") == 0)
+    force_ssl = 1;
+  else if (strcasecmp(value, "false") == 0)
+    force_ssl = 0; // Should already be set to 0 from initialization
+  else{
```

Comment:
space before the brace. If you format the code with `clang-format -style=LLVM` it should do this for you. See also https://collectd.org/wiki/index.php/Repository#Automatic_formatting